### PR TITLE
:bug: fix bit manipulation desription

### LINKED ISSUE #5: Bit Manipulation: what happens when indexes are out-of-bounds?
This issue relates to Section 14.3.1 "Bit Manipulation Operations" of the RI5CY User Manual.  Most of the bit manipulation instructions have two operands to select a slice from rs1.  For obvious reasons reasons the sum of these operands must be less than or equal to 32.

What happens if the sum is greater than 32?

### PR DESCRIPTION
**Pull request by davideschiavone**
This should fix #5

### CODE REVIEW

File: cores/cv32e40p/user_manual/source/instruction_set_extensions.rst [hunk 1]
```diff
@@ -317,6 +317,16 @@ together.
 -  Immediate branching instructions are useful to compare a register
    with an immediate value before taking or not a branch, see see :ref:`pulp_immediate_branching`.
 
+Extract, Insert, Clear and Set instructions have the following meaning:
+
+- Extract Is3+1/rs2[9:5]+1 bits from position Is2/rs2[4:0] [and sign extend it]
```
Comment (Silabs-ArjanB) [inline cores/cv32e40p/user_manual/source/instruction_set_extensions.rst:322]: Can you use () brackets to make the expression more clear? Same remark for below expressions

File: cores/cv32e40p/user_manual/source/instruction_set_extensions.rst [hunk 2]
```diff
@@ -317,6 +317,16 @@ together.
 -  Immediate branching instructions are useful to compare a register
    with an immediate value before taking or not a branch, see see :ref:`pulp_immediate_branching`.
 
+Extract, Insert, Clear and Set instructions have the following meaning:
```
Comment (Silabs-ArjanB) [inline cores/cv32e40p/user_manual/source/instruction_set_extensions.rst:320]: The file seems to use non-standard characters; see image below when I run make html:

image

File: cores/cv32e40p/user_manual/source/instruction_set_extensions.rst (hunk 1 above)
Comment (davideschiavone) [reply cores/cv32e40p/user_manual/source/instruction_set_extensions.rst]: done

File: cores/cv32e40p/user_manual/source/instruction_set_extensions.rst (hunk 2 above)
Comment (davideschiavone) [reply cores/cv32e40p/user_manual/source/instruction_set_extensions.rst:320]: I don't have such errors, can you help me to understand what to change?

**Comment by davideschiavone** [comment]
Can you please tell me what is not clear in the expressions? We can write them in many ways, like more verbose or more mathematical etc. What would you suggest?

**Pull request merged**